### DISCUSSION TITLE
Issue with vm.startBroadcast during Foundry test

### DISCUSSION THREAD

**Discussion by pherm0007** [Cyfrin/foundry-full-course-cu#2889] [Q&A]
Hi everyone,

I’m encountering an issue while running tests in my Foundry project. The error I’m getting is:

<img width="979" alt="Screenshot 2024-11-20 at 10 33 46" src="https://github.com/user-attachments/assets/779096ea-fafc-4708-9087-34a838e0a2a6">
 have a test setup where I’m using a deployment script to deploy contracts before running my tests. Here’s the relevant part of my code
<img width="1680" alt="Screenshot 2024-11-20 at 10 35 37" src="https://github.com/user-attachments/assets/ffacbb86-8f3a-407e-b194-feb7b2624df1">

Deployment Script:
<img width="1680" alt="Screenshot 2024-11-20 at 10 36 33" src="https://github.com/user-attachments/assets/e41a3929-fc34-4b6b-8b48-f98d3ac2917e">

**Comment by EngrPips** [comment]
Hello @pherm0007, Please run the test with `-vvvv` flag and share the stack trace of the test process.

**Reply by pherm0007** [reply]
<img width="1317" alt="Screenshot 2024-11-20 at 10 47 27" src="https://github.com/user-attachments/assets/c4a9e70c-fb94-43ac-8f3c-9728984e9e5a">
here is it

**Reply by EngrPips** [reply]
It seems like you started a broadcast in your `HelperConfig` and forgot to close it. Please confirm that.

**Reply by pherm0007** [reply]
oh thanks saw it and i changed it its working now

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
